app/jobs/shift_commerce/application_job.rb

```diff
@@ -0,0 +1,4 @@
+module Shiftcommerce
+  class ApplicationJob < ActiveJob::Base
+  end
+end
```

Comment:
Oops. I removed the mailer, but looks like removing the job got lost while I was mucking around. – will do that now.